### PR TITLE
CNV-5092 Guest agent information in the web console

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is associated with https://issues.redhat.com/browse/CNV-5092.

### CODE REVIEW

**Comment by lmandavi** [comment]
@pcbailey @yaacov - please review:
https://cnv-5092-guest-agent--ocpdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-web.html

**Comment by yaacov** [comment]
LGTM

**Comment by yfrimanm** [comment]
LGTM

File: modules/virt-about-qemu-guest-agent-web.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * .adoc
+
+[id="About-the-qemu-guest-agent-web_{context}"]
+= About the QEMU guest agent information in the web console
+
+When the QEMU guest agent is installed, the *Details* pane within the *Virtual machine overview* tab displays information about the *Operating system*, *Time zone*, and *Logged in users*.
```
Comment (pcbailey) [inline modules/virt-about-qemu-guest-agent-web.adoc:8]: The guest agent also provides the hostname of the VM.

File: modules/virt-about-qemu-guest-agent-web.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+The *Virtual machine overview* tab shows information about the guest operating system that was retrieved and displayed by the QEMU guest agent. If the QEMU guest agent is not installed, the *Virtual Machine Overview* tab displays information about the operating system that was specified when the virtual machine was created.
```
Comment (pcbailey) [inline modules/virt-about-qemu-guest-agent-web.adoc:10]: > about the guest operating system that was retrieved and displayed by the QEMU guest agent.

This language doesn't sound quite right. I would use something like "about the guest operating system installed on the virtual machine."

File: modules/virt-about-qemu-guest-agent-web.adoc (hunk 1 above)
Comment (lmandavi) [reply modules/virt-about-qemu-guest-agent-web.adoc:8]: Included Hostname as one of the items displayed in the tab.

File: modules/virt-about-qemu-guest-agent-web.adoc (hunk 2 above)
Comment (lmandavi) [reply modules/virt-about-qemu-guest-agent-web.adoc]: Updated the content.

**Comment by pcbailey** [comment]
LGTM

File: modules/virt-about-qemu-guest-agent-web.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-information.adoc
+
+[id="About-the-qemu-guest-agent-web_{context}"]
+= About the QEMU guest agent information in the web console
+
+When the QEMU guest agent is installed, the *Details* pane within the *Virtual machine overview* tab and the *Details* tab display information about the *Hostname*, *Operating system*, *Time zone*, and *Logged in users*.
+
```
Comment (gouyang) [inline modules/virt-about-qemu-guest-agent-web.adoc:9]: *Virtual machine overview* should be *Virtual Machine Overview*

File: modules/virt-about-qemu-guest-agent-web.adoc [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+The *Virtual machine overview* shows information about the guest operating system installed on the virtual machine.
```
Comment (gouyang) [inline modules/virt-about-qemu-guest-agent-web.adoc:10]: same here

File: modules/virt-about-qemu-guest-agent-web.adoc [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+The *Details* tab displays a table for *Logged in users.* This information includes *User name*, *Domain*, *Log in time*, and *Elapsed log in time*.
```
Comment (gouyang) [inline modules/virt-about-qemu-guest-agent-web.adoc:12]: It's "Login time" and "Elapsed logged in time".

Comment (lmandavi) [reply modules/virt-about-qemu-guest-agent-web.adoc:12]: Contacted @pcbailey with proposed changes to these terms.

Comment (gouyang) [reply modules/virt-about-qemu-guest-agent-web.adoc]: `Log in time` should be `Login time`.
Others look good to me.

**Comment by lmandavi** [comment]
@gouyang Please review and advise if this PR can move to peer review.

**Comment by gouyang** [comment]
LGTM

File: modules/virt-about-qemu-guest-agent-web.adoc [hunk 6]
```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-information.adoc
+
+[id="About-the-qemu-guest-agent-web_{context}"]
+= About the QEMU guest agent information in the web console
+
+When the QEMU guest agent is installed, the *Details* pane within the *Virtual Machine Overview* tab and the *Details* tab display information about the *Hostname*, *Operating system*, *Time zone*, and *Logged in users*.
```
Comment (bergerhoffer) [inline modules/virt-about-qemu-guest-agent-web.adoc:8]: This section is a little overwhelming with all of the bolded GUI items. It totally makes sense when you're talking about "... the **Details** pane within the **Virtual Machine Overview** tab ...", but I think that you could list the details that are displayed without actually providing bolded GUI items.

```
When the QEMU guest agent is installed, the *Details* pane within the *Virtual Machine Overview* tab and the *Details* tab display information about the host name, operating system, time zone, and logged in users.
```

File: modules/virt-about-qemu-guest-agent-web.adoc [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+The *Virtual Machine Overview* shows information about the guest operating system installed on the virtual machine.
+
+The *Details* tab displays a table for *Logged in users.* This information includes *User name*, *Domain*, *Login time*, and *Elapsed Logged in time*.
```
Comment (bergerhoffer) [inline modules/virt-about-qemu-guest-agent-web.adoc:12]: Same comment about reducing the number of actual GUI items shown.

```
 The *Details* tab displays a table for logged in users. This information includes the user name, domain, login time, and elapsed logged in time.
```

Though I might question if it's really necessary to call out all of the fields listed in the table.

File: modules/virt-about-qemu-guest-agent-web.adoc [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+The *Disks* tab displays a table for *File systems*. This information includes *Name*, *File system type*, *Mount point*, *Total bytes*, and *Used bytes*.
```
Comment (bergerhoffer) [inline modules/virt-about-qemu-guest-agent-web.adoc:14]: Same comment about the GUI items.

```
 The *Disks* tab displays a table for file systems. This information includes the name, file system type, mount point, total bytes, and used bytes.
```

And whether it's really necessary to call out the fields.

File: modules/virt-about-qemu-guest-agent-web.adoc [hunk 9]
```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-viewing-qemu-guest-agent-information.adoc
+
+[id="About-the-qemu-guest-agent-web_{context}"]
+= About the QEMU guest agent information in the web console
+
+When the QEMU guest agent is installed, the *Details* pane within the *Virtual Machine Overview* tab and the *Details* tab display information about the *Hostname*, *Operating system*, *Time zone*, and *Logged in users*.
+
+The *Virtual Machine Overview* shows information about the guest operating system installed on the virtual machine.
```
Comment (bergerhoffer) [inline modules/virt-about-qemu-guest-agent-web.adoc:10]: Is **Virtual Machine Overview** a tab or a page, or what?

**Comment by bergerhoffer** [comment]
Just need to know what versions this applies to, and then also commits need to be squashed.

File: modules/virt-about-qemu-guest-agent-web.adoc (hunk 6 above)
Comment (lmandavi) [reply modules/virt-about-qemu-guest-agent-web.adoc]: Resolved by removing bolded items and limiting the mention of the GUI items.

File: modules/virt-about-qemu-guest-agent-web.adoc (hunk 8 above)
Comment (lmandavi) [reply modules/virt-about-qemu-guest-agent-web.adoc]: Resolved by removing the bolded items, and mentioning only the table for file system.s

**Comment by lmandavi** [comment]
Applies to enterprise 4.6. Commits were squashed.

**Review by bergerhoffer — COMMENTED** [review]
This LGTM now. Once the build passes (they're backed up right now), then this can be merged and cherry-picked.

File: modules/virt-about-qemu-guest-agent-web.adoc (hunk 9 above)
Comment (lmandavi) [reply modules/virt-about-qemu-guest-agent-web.adoc]: Tab.

**Pull request merged**